keps/sig-cloud-provider/20190422-cloud-controller-manager-migration.md

```diff
@@ -136,6 +137,7 @@ inside the KCM (see Figure 1). As a result, in v1.17 those controllers would run
 To migrate to the CCM for v1.18, the cloud provider would update the `cloud-network-controller-migration` lock to now run in the CCM (see Figure 2).
 During a control plane upgrade, the cloud network controllers may still run in one of the KCMs that are still on v1.17. A 1.17 KCM holding the lock
 will prevent any of the v1.18 CCMs from claiming the lock. When the current holder of the lock goes down, one of the controller managers eligible will acquire lock.
+Then in 1.19 the cloud provider would delete the `cloud-network-controller-migration` lock and run using the `cloud-controller-manager` lock for the migrated controllers.
```

Comment:
Worth noting in the 1.18 step that the `cloud-network-controller-migration` would no longer be in the KCM.